hikari/events/reaction_events.py

```diff
@@ -216,6 +233,23 @@ def emoji_id(self) -> typing.Optional[snowflakes.Snowflake]:
             `builtins.None`.
         """
 
+    def is_for_emoji(self, emoji: typing.Union[emojis.Emoji, str], /) -> bool:
+        """Get whether the reaction event is for a specific emoji.
+
+        Parameters
+        ----------
+        emoji : typing.Union[hikari.emojis.Emoji, builtins.str]
+            The emoji to check.
+
+            Passing `builtins.str` here indicates a unicode emoji.
+
+        Returns
+        -------
+        builtins.bool
+            Whether the emoji is the one which was removed.
+        """
+        return emoji.id == self.emoji_id if isinstance(emoji, emojis.CustomEmoji) else emoji == self.emoji_name
```

Comment:
This isnt exactly correct, as passing `None` as the emoji will return `True` if Discord dont send the emoji_name.

Reply:
It doesn't document any support for passing None

Reply:
Fair enough